test/e2e/cpulist_test.go

```diff
@@ -0,0 +1,149 @@
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+)
+
+var _ = g.Describe("cpulist", func() {
+	g.Context("with arguments", func() {
+		g.It("parses correctly commandline", func() {
+			cmdline := []string{
+				filepath.Join(binariesPath, "cpulist"),
+				"-c",
+				"0-3,5,7-9",
+			}
+			fmt.Fprintf(g.GinkgoWriter, "running: %v\n", cmdline)
+
+			cmd := exec.Command(cmdline[0], cmdline[1:]...)
+			cmd.Stderr = g.GinkgoWriter
```

Comment:
What's the purpose of this line where it used?

Reply:
we redirect all the error messages to ginkgo, so we see them for free if the tests fail. Or at least that's the intention :)